asabiris/output/slack/service.py

```diff
@@ -72,4 +72,4 @@ async def send(self, body: str, atts: List[Tuple[bytes, str, str]]) -> None:
 						initial_comment=initial_comment
 					)
 		except SlackApiError as e:
-			raise SlackApiError("Failed to send message: {}".format(e))
+			raise SlackApiError("Error sending Slack message: " + e.response['error'], 401)
```

Comment:
because of this

  File "/home/franta/projects/asab-iris/asabiris/output/slack/service.py", line 75, in send
    raise SlackApiError("Failed to send message: {}".format(e))
TypeError: __init__() missing 1 required positional argument: 'response'


**SlackApiError** class is being instantiated without providing the required positional argument called **response**.